### ISSUE TITLE
Train on new dataset

### ISSUE THREAD

**Issue by wenxichen** [endernewton/tf-faster-rcnn#17]
Hi,

Great project. I have successfully ran the training and testing. I am wondering what do I need to change to train on a new dataset?

Thanks!

**Comment by philokey** [comment]
You need to add a new class of you dataset like [pascal_voc.py](https://github.com/endernewton/tf-faster-rcnn/blob/master/lib/datasets/pascal_voc.py)

**Comment by wenxichen** [comment]
Great. Sounds good. Thanks

**Comment by zdm123** [comment]
@wenxichen hi， did you train a model on your own dataset. Can you give me some advices about that? Do I need to download a pre-trained model>

**Comment by wenxichen** [comment]
@zdm123 You can take a look at my fork at https://github.com/wenxichen/tf-faster-rcnn. More specifically here (https://github.com/wenxichen/tf-faster-rcnn/blob/master/lib/datasets/mio_tcd_loc.py) is another dataset class I have created for MIO-TCD dataset.
After you created a dataset classes for your own dataset, you can incorporated it into the training script (https://github.com/wenxichen/tf-faster-rcnn/blob/master/experiments/scripts/train_faster_rcnn.sh). There are a few places you need to change to make the entire flow work, but that detail you can find from the error messages when you trying to run the training.
My fork is from couple months ago, so might not have as much features as it has now. But the tf faster rcnn training and detection should be good.

**Comment by jiaxinying** [comment]
What if my own dataset contains labels that not include in original?
